assets/shaders/post_process_pass.wgsl

```diff
@@ -0,0 +1,32 @@
+// This shader computes the chromatic aberration effect
+
+#import bevy_pbr::utils
+
+// Since post process is a fullscreen effect, we use the fullscreen vertex stage from bevy
```

Comment:
I feel like we should explain "import the vertex stage" more, and maybe a short sentence or two on how the fullscreen thing is essentially helpers for drawing a single large triangle covering the entire screen.

Reply:
I tried making an ascii triangle for that and explaining it a bit more, but I didn't manage to have something that looked good. I'll try with just words I guess.